### PR TITLE
[IMP] sale_margin_percentage: make purchase price editable I#26616

### PR DESCRIPTION
**Pull request by xmglord**
This module was originally created in v11 for a specific customer, and
the purchase price needed to be non-editable for some reason. Now the
purchase price does not need to be non-editable. If it is needed, it
should be managed in the view because the field is natively editable.

### CODE REVIEW

**Comment by luisg123v** [comment]
> Since this customer is now in v15, the purchase price does not need to be non-editable.

I don't see the cause-effect relation here.

**Comment by xmglord** [comment]
> > Since this customer is now in v15, the purchase price does not need to be non-editable.
> 
> I don't see the cause-effect relation here.

It was not ready :c 
Now please review @luisg123v

File: sale_margin_percentage/models/sale_order_line.py [hunk 1]
```diff
@@ -7,7 +7,7 @@ class SaleOrderLine(models.Model):
     margin_threshold = fields.Float(
         default=lambda self: self.env.user.company_id.margin_threshold, help="Limit margin set in sales configuration"
     )
-    purchase_price = fields.Float(readonly=True, help="Price purchase of product")
+    purchase_price = fields.Float(help="Price purchase of product")
```
Comment (luisg123v) [inline sale_margin_percentage/models/sale_order_line.py]: Just remove inheritance

**Review by luisg123v — CHANGES_REQUESTED** [review]
Update PR's description to match commit's description

File: sale_margin_percentage/models/sale_order_line.py (hunk 1 above)
Comment (xmglord) [reply sale_margin_percentage/models/sale_order_line.py]: done

**Review by luisg123v — APPROVED** [review]

**Pull request merged**